lib/rucio/core/volatile_replica.py

```diff
@@ -32,14 +32,13 @@
 
 
 @transactional_session
-def add_volatile_replicas(rse_id, replicas, *, session: "Session"):
+def add_volatile_replicas(rse_id: str, replicas: list[dict[str, Any]], *, session: "Session") -> None:
```

Comment:
The only use of `replicas` is in:

```python
for replica in replicas:
```

I wonder if using `typing.Sequence` is a better practise for *function parameters*, when there’s nothing `list`-specific. But I do not propose to change this now, just food for thought.

Reply:
That's a good point, I agree. I'll make this change now so that I can also amend the commit message to address the other comment

Reply:
Forgive me, I gave you bad advice. `typing.Sequence` is deprecated and must be replaced with `collections.abc.Sequence`.

Reply:
I’m now even more conflicted because `collections.abc.Iterable` might be more accurate.

Reply:
I agree (relevant thread: https://stackoverflow.com/questions/72157296/what-is-the-difference-between-type-hinting-a-variable-as-an-iterable-versus-a-s)

Amended in latest revision